components/builder-http-gateway/src/http/middleware.rs

```diff
@@ -204,7 +206,11 @@ impl BeforeMiddleware for Authenticated {
                         // Check to see if this is a valid github token, and create (or
                         // update) a session. This is a temporary fix until we can roll out
                         // and migrate clients to our own personal access tokens.
-                        session_create_github(req, token)?
+                        if env::var_os("HAB_FUNC_TEST").is_some() {
```

Comment:
In my original PR, I had a lengthy comment in this file explaining the nature of our session tokens and why our current implementation was broken but it seems to have disappeared.  Any idea why or where it went to?